.ci/msys_test.bat

```diff
@@ -0,0 +1,7 @@
+:: FIXME(equalsraf) currently not all tests work on Windows
+:: for now just run what we can
+mingw32-make functionaltest TEST_FILE=test\functional\eval\printf_spec.lua VERBOSE=1 || goto :error
```

Comment:
You mean tag describe() with `#windows` as we verify that each test runs on windows. Might be a bother later on, when almost all tests would be tagger, but its a good way to go about it. TEST_FILTER might also work.

I'm trying to get this to work but TEST_TAG=windows doesn't seem to have any effect (no tests are run). I assumed the only thing I had to do was add `#windows` in describe but maybe I was wrong.